### PR TITLE
Fixed #10 thanks help of @hanyouqing

### LINKED ISSUE #10: hugo server image relative path issue
Current the markdown relative path doesn't work for the hugo server, it's quite difficult for us support markdown display and hugo server rendering at the same time.
Maybe we can find a way to support both senairons at same time.

### PR DESCRIPTION
**Pull request by WillemJiang**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -3,4 +3,4 @@ languageCode = "en-us"
 title = "ALC Beijing"
 disableKinds = ["taxonomy", "taxonomyTerm"]
 theme = "tale"
-canonifyURLs = true
```
Comment (kezhenxu94) [inline config.toml:6]: But why remove this, it breaks the build

Comment (WillemJiang) [reply config.toml:6]: To fix the image display issue, we need to set uglyurls = true.

Comment (kezhenxu94) [reply config.toml:6]: > To fix the image display issue, we need to set uglyurls = true.

Can't `canonifyURLs` and `uglyurls` exist simultaneously?

Comment (WillemJiang) [reply config.toml]: I just find another to workaround the relative path issue.

**Pull request merged**